mmengine/optim/scheduler/param_scheduler.py

```diff
@@ -1130,10 +1144,10 @@ class CosineRestartParamScheduler(_ParamScheduler):
         periods (list[int]): Periods for each cosine anneling cycle.
         restart_weights (list[float]): Restart weights at each
             restart iteration. Defaults to [1].
-        eta_min (float): Minimum parameter value at the end of scheduling.
-            Defaults to None.
+        eta_min (float, optional): Minimum parameter value at the end of
+            scheduling. Defaults to None.
         eta_min_ratio (float, optional): The ratio of minimum parameter value
-            to the base parameter value. Either `min_lr` or `min_lr_ratio`
+            to the base parameter value. Either `eta_min` or `eta_min_ratio`
             should be specified. Default: None.
```

Comment:
```suggestion
            to the base parameter value. Either `eta_min` or `eta_min_ratio`
            should be specified.
```

Reply:
I dont understand why delete `Default: None` here

Reply:
type hint `Optional` means a default value with None.

Reply:
Besides, add `New in version 0.4.0`